platform/windows/gl_manager_windows_native.cpp

```diff
@@ -323,45 +343,45 @@ Error GLManager_Windows::_create_context(GLWindow &win, GLDisplay &gl_display) {
 	}; //zero indicates the end of the array
 
 	PFNWGLCREATECONTEXTATTRIBSARBPROC wglCreateContextAttribsARB = nullptr; //pointer to the method
-	wglCreateContextAttribsARB = (PFNWGLCREATECONTEXTATTRIBSARBPROC)wglGetProcAddress("wglCreateContextAttribsARB");
+	wglCreateContextAttribsARB = (PFNWGLCREATECONTEXTATTRIBSARBPROC)gd_wglGetProcAddress("wglCreateContextAttribsARB");
 
 	if (wglCreateContextAttribsARB == nullptr) //OpenGL 3.0 is not supported
 	{
-		wglDeleteContext(gl_display.hRC);
+		gd_wglDeleteContext(gl_display.hRC);
 		gl_display.hRC = 0;
 		return ERR_CANT_CREATE;
 	}
 
 	HGLRC new_hRC = wglCreateContextAttribsARB(win.hDC, 0, attribs);
 	if (!new_hRC) {
-		wglDeleteContext(gl_display.hRC);
+		gd_wglDeleteContext(gl_display.hRC);
 		gl_display.hRC = 0;
 		return ERR_CANT_CREATE;
 	}
 
-	if (!wglMakeCurrent(win.hDC, nullptr)) {
+	if (!gd_wglMakeCurrent(win.hDC, nullptr)) {
 		ERR_PRINT("Could not detach OpenGL context from newly created window: " + format_error_message(GetLastError()));
 	}
 
-	wglDeleteContext(gl_display.hRC);
+	gd_wglDeleteContext(gl_display.hRC);
 	gl_display.hRC = new_hRC;
 
-	if (!wglMakeCurrent(win.hDC, gl_display.hRC)) // Try To Activate The Rendering Context
+	if (!gd_wglMakeCurrent(win.hDC, gl_display.hRC)) // Try To Activate The Rendering Context
```

Comment:
```suggestion
	if (!gd_wglMakeCurrent(win.hDC, gl_display.hRC)) // Try To Activate The Rendering Context.
```

Reply:
Could Also Remove The Title Case Which Doesn't Make Sense For a Comment, but I think this is a pre-existing style in this file and maybe not worth doing in this PR.